.idea/.name

```diff
@@ -0,0 +1 @@
+MilaPetrova-app
```

Comment:
do not include any computer generated files e.g. 
./idea 
target 
*.xmi
etc. 
Review this week's chat for info here.